src/plugins/usage_collection/server/report/schema.ts

```diff
@@ -43,5 +43,6 @@ export const reportSchema = schema.object({
   application_usage: schema.maybe(schema.recordOf(schema.string(), applicationUsageReportSchema)),
 });
 
+// should the schema types also be versioned explicitly?
```

Comment:
Good question, for now I think it is OK not to. But it would be consistent with our types. I could go either way on this one 😁

Reply:
> or now I think it is OK not to

Cool! I'm going with that :-)